src/store/sandbox/actions.ts

```diff
@@ -87,8 +90,10 @@ export const fetchAccount = createAsyncThunk<AccountPayload, string>(
 export const requestFaucetFunds = createAsyncThunk<Account, string>(
   'account/faucet',
   // @ts-ignore
-  async (principal, { rejectWithValue }) => {
-    const res = await postToSidecar(`/debug/faucet?address=${principal}`);
+  async (principal, { rejectWithValue, getState }) => {
+    // @ts-ignore
```

Comment:
🙀

Reply:
also same issue https://github.com/blockstack/explorer/pull/134/files/878ab1d54796294bb3e53a0cd3923367e0b8345e#r426730594

Reply:
The type error here is helpful. The typing says it returns `Account`, but it only returns two of Account's 4 props:

```
export interface Account {
    balance: BN;
    nonce: number;
}
```

modified above